test/modules/string/replace_test.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flodash/flodash.dart' as flodash;
+
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('replace', () {
+    test('replace should replace all occurrences of pattern in the string', () {
+      String input = 'Hello, Hello, Hello!';
+      String expected = 'Hi, Hi, Hi!'; // Update the expected result
```

Comment:
This comment can be safely removed I guess, the variable name is self explanatory.